src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -89,4 +90,52 @@ Here are the available checks alongside the insight they offer:
+
+## Mail Processing
+
+Mail processing allows to take asynchronously business decisions on received emails.
+
+Here are it's components:
+
+ - The `spooler` takes mail out of the mailQueue and executes mail processing within the `mailet container`.
+ - The `mailet container` synchronously execute the user defined logic. This 'logic' is written though the use of `mailet`,
+ `matcher` and `processor`.
+ - A `mailet` represent an action: mail modification, envelop modification, a side effect, or stop processing.
+ - A `matcher` represent a condition to execute a mailet.
+ - A `processor` is a flow of pair of `matcher` and `mailet` executed sequentially. The `ToProcessor` mailet is a `goto` 
+ instruction to start executing another `processor`
+ - A `mail repository` allow storage of a mail as part of its processing. Standard configuration relies on the following 
+ mail repository:
+     - `cassandra://var/mail/error/` : unexpected errors that occurred during mail processing. Emails impacted by performance 
+     related exceptions, or logical bug within James code are typically stored here. These mail could be reprocessed once the 
+     cause of the error is fixed. The `Mail::error` field can help diagnose the issue. Correlation with logs can be achieved
+     via the use of the `Mail::name` field.
+     - `cassandra://var/mail/address-error/` : mail addressed to a non existing recipient of an handled local domain. These 
+     mails could be reprocessed once the user is created, for instance.
+     - `cassandra://var/mail/relay-denied/` : mail for whom relay was denied: missing authentication can for instance be a 
+     cause. In addition to prevent disasters upon miss configuration, an email review of this mail repository can help
+     refine a host spammer blacklist.
+     - `cassandra://var/mail/rrt-error/` : runtime error upon Recipient Rewritting occurred. This is typically due to a loop.
+     We recommend verifying user mappings via [User Mappings webadmin API](manage-webadmin.html#user-mappings) then once identify 
+     break the loop by removing some Recipient Rewrite Table entry via the 
+     [Delete Alias](manage-webadmin.html#removing-an-alias-of-an-user), [Delete Group member](manage-webadmin.html#removing-a-group-member), 
+     [Delete forward](manage-webadmin.html#removing-a-destination-of-a-forward), [Delete Address mapping](manage-webadmin.html#remove-an-address-mapping), 
+     [Delete Domain mapping](manage-webadmin.html#removing-a-domain-mapping) or 
+     [Delete Regex mapping](manage-webadmin.html#removing-a-regex-mapping) APIs (as needed). The `Mail::error` field can 
+     help diagnose the issue as well. Then once the root cause adressed, the mail can be reprocessed.
+
+Read [this](config-mailetcontainer.html) to discover mail processing configuration, including error management.
+
+Currently, an administrator can monitor mail processing failure through `ERROR` log review. We also recommend watching in Kibana
```

Comment:
Kibana? First time I see it there (but I did not take the PRs in the right order). Is it documented elsewhere or is it Linagora-related?

Reply:
Line 42 of this file `Structured logging into Kibana`